sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/HiveTableScanExec.scala

```diff
@@ -60,8 +59,6 @@ case class HiveTableScanExec(
   require(partitionPruningPred.isEmpty || relation.isPartitioned,
     "Partition pruning predicates only supported for partitioned tables.")
 
-  override def conf: SQLConf = sparkSession.sessionState.conf
```

Comment:
This actually makes sense, to make sure the conf matches the spark session. How about we move this override to `SparkPlan`?

Reply:
reverted